src/components/UserPortal/ChatRoom/MessageInput.tsx

```
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Button, InputGroup, Form } from 'react-bootstrap';
```

ESLint

[error] 'Button' import from 'react-bootstrap' is restricted. Direct imports of Button from react-bootstrap are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)